build/Dockerfile

```diff
@@ -1,8 +1,21 @@
 FROM quay.io/sustainable_computing_io/kepler_builder:ubi-9-libbpf-1.3.0 AS builder
+ARG INSTALL_DCGM=false
+ARG INSTALL_HABANA=false
 WORKDIR /workspace
 
 COPY . .
 
+RUN if [[ "$INSTALL_DCGM" == "true" ]]; then \
+	dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel9/x86_64/cuda-rhel9.repo; \
+	yum install -y datacenter-gpu-manager; \
+fi;
+
+RUN if [[ "$INSTALL_HABANA" == "true" ]]; then \
+	rpm -Uvh https://vault.habana.ai/artifactory/rhel/9/9.2/habanalabs-firmware-tools-1.15.1-15.el9.x86_64.rpm --nodeps; \
+	echo /usr/lib/habanalabs > /etc/ld.so.conf.d/habanalabs.conf; \
+	ldconfig; \
+fi;
+
```

Comment:
How is it different from this? https://github.com/sustainable-computing-io/kepler/pull/1785/files#diff-5aecd73bbcb9b7410938673269f0d4e676dd18509277b0ff2f2787c9d3e22604R44